src/main/java/com/tagtraum/perf/gcviewer/action/OpenFile.java

```diff
@@ -60,6 +63,19 @@ public OpenFile(final GCViewerGui gcViewer) {
 
     public void actionPerformed(final ActionEvent e) {
         final boolean aDocumentIsAlreadyOpen = gcViewer.getSelectedGCDocument() != null;
```

Comment:
I don't think it a particularly good idea to default to "add" if a document is already opened. That's not the default in the current implementation, either. I suggest to default to opening a separate window, because that seems to be the more defensive option to me.